fiftyone/operators/panel.py

```diff
@@ -213,12 +213,10 @@ def set(self, key, value=None):
         """
         d = key if isinstance(key, dict) else {key: value}
 
-        args = {}
         for k, v in d.items():
             super().set(k, v)
```

Comment:
Looks like this can be further simplified as:

```suggestion
        super().set(d)
```